src/main/java/org/alliancegenome/curation_api/model/entities/Reference.java

```diff
@@ -72,20 +60,46 @@ public class Reference extends InformationContentEntity {
 	@Transient
 	@JsonIgnore
 	public String getReferenceID() {
+		return getReferenceID(true);
+	}
+
+	/**
+	 * Retrieve PUB MOD ID
+	 */
+	@Transient
+	@JsonIgnore
+	public String getPubModID() {
+		return getReferenceID(false);
+	}
+
+	@Transient
+	private String getReferenceID(boolean pubmedIdFirst) {
```

Comment:
Seems like this method does two different functions based on boolean input. I would guess this should be two methods?

Reply:
Yes, that's why it's marked private. The two public methods calling this method are defined as 
getReferenceID()
and
getPubModID()